asab/metrics/influxdb.py

```diff
@@ -196,11 +196,33 @@ def metric_to_influxdb(metric_record, now):
 			# SKIP empty fields
 			if not field.get("values") or field.get("values") == {}:
 				continue
-			values_lines.append(build_metric_line(field.get("tags"), field.get("values")))
+			values_lines.append(build_metric_line(field.get("tags"), (field.get("values"))))
 
 	return ["{},{} {}\n".format(name, line, int(timestamp * 1e9)) for line in values_lines]
 
 
+def validate_name(name: str):
+	return name.replace(" ", "\\ ").replace(",", "\\,")
+
+
+def cleanup_tags(tags: dict):
```

Comment:
Please add some kind of document string to explain what you cleaning up here.

The same for other function bellow.

Reply:
Changed cleanup_tags to escape_tags, because that's what it does, and added a docstring